test/controllers/session_controller_test.exs

```diff
@@ -106,8 +105,7 @@ defmodule ApiWeb.SessionControllerTest do
           "display_name" => "#{user.first_name} #{user.last_name}",
           "gravatar_hash" => UserHelper.gravatar_hash(user),
           "role" => "owner"
-        }],
-        "project" => nil
+        }]
```

Comment:
Let's add a dangling comma to minimize diffs in the future.

Can we teach Credo to have opinions on this?

Reply:
I don't need this solved to ship. Maybe it'd be even better if we dealt with this in a different commit?

Reply:
I agree with creating a new task for this. Just added it to trello

Reply:
:+1:

Reply:
Card here: https://trello.com/c/6Rs8Rphl